bundle/language_sw.properties

```diff
@@ -1,1480 +1,1493 @@
+angal.exa.neweditresult.title                                                    = Mpya/kuhariri matokeo
+angal.exa.results.fmt.title                                                      = {0} Results
+angal.exatype.examtypebrowser.title                                              = Kipitiapitia aina za uchunguzi
+angal.exatype.newexamtype.title                                                  = Aina mpya ya uchunguzi
+angal.exatype.editexamtype.title                                                 = kuhariri aina ya matokeo
+angal.hospital.hospitalinformation.title                                         = Hospital Information
+angal.lab.laboratorybrowser.title                                                = Laboratory Browser
+angal.lab.newlaboratoryexam.title                                                = Uchunguzi mpya wa maabara
+angal.lab.editlaboratoryexam.title                                               = Kuhariri Uchunguzi wa maabara
+angal.login.title                                                                = Login
+angal.mainmenu.fmt.title                                                         = User: {0}
+angal.messagedialog.error.title                                                  = Error
+angal.messagedialog.warning.title                                                = Warning
+angal.messagedialog.info.title                                                   = Information
+angal.messagedialog.question.title                                               = Question
+angal.messagedialog.plain.title                                                  = Note
+angal.userbrowser.title                                                          = User Browser
+angal.userbrowser.addnewuser.title                                               = Add New User
+angal.userbrowser.edituser.title                                                 = Edit User
+angal.userbrowser.resetpassword.title                                            = Reset Password
+angal.groupsbrowser.title                                                        = Groups Browser
+angal.groupsbrowser.newgroup.title                                               = New Group
+angal.groupsbrowser.editgroup.title                                              = Edit Group
+angal.groupsbrowser.menuitembrowser.title                                        = Menu Item Browser
+angal.malnutrition.malnutritionbrowser.title                                     = Kupitiapitia utapiaji mlo
+angal.malnutrition.newmalnutrition.title                                         = New Malnutrition
+angal.malnutrition.editmalnutrition.title                                        = Edit Malnutrition
+angal.medicals.newmedical.title                                                  = New Medical
+angal.medicals.editmedical.title                                                 = Edit Medical
+angal.medicals.pharmaceuticalbrowser.title                                       = Pharmaceutical Browser
```

Comment:
If the key is missing, OH should log the missing key and should fallback to English (we have still to understand if and how to report it also in the GUI) but in such case the key is not missing and the translation is same as English, so OH will not report anything.

Is this the intended behavior?

Reply:
All the files have every key defined; if there isn't a translation then the English version of the string is used in the file.  This means every key should be found in every file.  I was hoping this would make translation easier and I noticed that some of the files when I started were a mix of languages.

Reply:
Thanks @dbmalkovsky, actually it is not a must that every file has all the keys, definitely it could be the signal that a translation is missing... and it is also what it is done by Transifex

I merged #579, #583 and #584 just to see if Transifex also checks if the translation is different or not from the default (English)